### PR TITLE
ensure #lesson_released? works for all cases

### PR DESCRIPTION
**Pull request by ericcf**
[#94530796]

### CODE REVIEW

File: spec/models/concerns/status_spec.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+require "spec_helper"
+
+module Concerns
+  class MockParticipant
+    include Status
+
+    attr_reader :study_day
+
+    def initialize(study_day:)
```
Comment (michaelwehrley) [inline spec/models/concerns/status_spec.rb]: Why don't you just use `attr_accessor` here since you are both reading and writing it?

Comment (michaelwehrley) [inline spec/models/concerns/status_spec.rb]: I see - you are making sure that a study_day is passed in :-)

Comment (michaelwehrley) [inline spec/models/concerns/status_spec.rb]: You will get "ArgumentError: missing keyword: study_day"

Comment (ericcf) [inline spec/models/concerns/status_spec.rb]: No strong reason. They seem like comparable approaches to me.

File: spec/models/concerns/status_spec.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    end
+  end
+
+  RSpec.describe Status do
+    describe "#lesson_released?" do
+      let(:started_participant) { MockParticipant.new(study_day: 3) }
+
+      context "when the lesson is nil" do
+        it "returns false" do
+          expect(started_participant.lesson_released?(nil)).to be_falsey
```
Comment (michaelwehrley) [inline spec/models/concerns/status_spec.rb]: Haven't you recommended in the past that we should be open to inputs but specific in outputs?  Why `be_falsey` here and not `eq(false)`?  Thanks!

Comment (ericcf) [inline spec/models/concerns/status_spec.rb]: Good point! Yeah I think I'll change that.

**Comment by michaelwehrley** [comment]
:eyeglasses: good!

**Pull request merged**